src/component/IconButton.tsx

```diff
@@ -0,0 +1,42 @@
+// This is a skeleton starter React component generated by Plasmic.
+// This file is owned by you, feel free to edit as you see fit.
+import * as React from "react";
+import {
+  PlasmicIconButton,
+  DefaultIconButtonProps
+} from "./plasmic/flip_card/PlasmicIconButton";
+
+// Your component props start with props for variants and slots you defined
+// in Plasmic, but you can add more here, like event handlers that you can
+// attach to named nodes in your component.
+//
+// If you don't want to expose certain variants or slots as a prop, you can use
+// Omit to hide them:
+//
+// interface IconButtonProps extends Omit<DefaultIconButtonProps, "hideProps1"|"hideProp2"> {
+//   // etc.
+// }
+//
+// You can also stop extending from DefaultIconButtonProps altogether and have
+// total control over the props for your component.
+interface IconButtonProps extends DefaultIconButtonProps {}
+
+function IconButton(props: IconButtonProps) {
```

Comment:
How does it now work when we want to structure the project our way? I would `src/component/IconButton.tsx` like to be `src/component/button/index.tsx` or something like that. I do not want to have 30 different files flying around in the same package.